eq-author/src/components/CommentsPanel/index.js

```diff
@@ -398,7 +390,11 @@ const CommentsPanel = ({ componentId, me: { id: myId } }) => {
 
   const displayComments = comments.map((item, index) => {
     const replies = comments[index].replies;
-    const displayReplies = replies.map((repliesItem, repliesIndex) => {
+    let sortedReplies = replies.slice() || [];
+    sortedReplies = sortedReplies.sort((a, b) =>
+      b.createdTime > a.createdTime ? 1 : -1
+    );
```

Comment:
```suggestion
    const sortedReplies = replies.slice() || [];
    sortedReplies = sortedReplies.sort((a, b) =>
      b.createdTime > a.createdTime ? 1 : -1
    );
```

This can be a const for the same reason as above.